lbrynet/core/Wallet.py

```diff
@@ -638,10 +638,16 @@ def _clean_bad_records(self):
         d = self.db.runQuery("delete from name_metadata where length(txid) > 64 or txid is null")
         return d
 
+
     def _save_name_metadata(self, name, txid, nout, sd_hash):
         assert len(txid) == 64, "That's not a txid: %s" % str(txid)
         d = self.db.runQuery("delete from name_metadata where name=? and txid=? and n=? and sd_hash=?",
                              (name, txid, nout, sd_hash))
+        # delete legacy v1 database entry with missing nouts
```

Comment:
I like to avoid comments that explain what the code is doing; I believe comments should be reserved for explanations of why the code is as it is.

If a comment is necessary, consider making changes to improve the readability of the code.  In this case, I'd make a function call: `d.addCallback(lambda _: self.delete_legacy_entries(name, txid, sd_hash))`